decidim-meetings/spec/system/report_meeting_spec.rb

```diff
@@ -2,21 +2,20 @@
 
 require "spec_helper"
 
-# REDESIGN_PENDING - Integrate flag in meeting show
-# describe "Report Meeting", type: :system do
-#   include_context "with a component"
+describe "Report Meeting", type: :system do
+  include_context "with a component"
 
-#   let(:manifest_name) { "meetings" }
-#   let!(:meetings) { create_list(:meeting, 3, :published, component:) }
-#   let(:reportable) { meetings.first }
-#   let(:reportable_path) { resource_locator(reportable).path }
-#   let!(:user) { create :user, :confirmed, organization: }
+  let(:manifest_name) { "meetings" }
+  let!(:meetings) { create_list(:meeting, 3, :published, component:) }
+  let(:reportable) { meetings.first }
+  let(:reportable_path) { resource_locator(reportable).path }
+  let!(:user) { create(:user, :confirmed, organization:) }
 
-#   let!(:component) do
-#     create(:proposal_component,
-#            manifest:,
-#            participatory_space: participatory_process)
-#   end
+  let!(:component) do
```

Comment:
Looking at this test, seems that it is implemented wrong, as the `meetings` is using a component, yet the component is a `proposal`. Nothing to do here, just need to extract a ticket.

Reply:
I've tried locally to change the component with meetings and the tests pass anyway